botocore/serialize.py

```diff
@@ -802,10 +1013,35 @@ def _default_serialize(self, xmlnode, params, shape, name):
         node.text = str(params)
 
 
+class RpcV2CBORSerializer(BaseRpcV2Serializer, CBORSerializer):
+    TIMESTAMP_FORMAT = 'unixtimestamp'
+
+    def _serialize_body_params(self, parameters, input_shape):
+        body = bytearray()
+        self._serialize_data_item(body, parameters, input_shape)
+        return bytes(body)
+
+    def _serialize_headers(self, serialized, operation_model):
+        serialized['headers']['smithy-protocol'] = 'rpc-v2-cbor'
+
+        if operation_model.has_event_stream_output:
+            header_val = 'application/vnd.amazon.eventstream'
+        else:
+            header_val = 'application/cbor'
+
+        has_body = serialized['body'] != b''
+        has_content_type = has_header('Content-Type', serialized['headers'])
+
+        serialized['headers']['Accept'] = header_val
+        if not has_content_type and has_body:
+            serialized['headers']['Content-Type'] = header_val
```

Comment:
```suggestion
            serialized['headers']['Content-Type'] = content_type
```

Reply:
[Spec](https://smithy.io/2.0/additional-specs/protocols/smithy-rpc-v2.html):
> Requests for the rpcv2Cbor protocol MUST use the following behavior for setting an Accept header:

>    For requests with event streaming responses: the value of the Accept
        header MUST be application/vnd.amazon.eventstream.

>    For requests with all other response types: the value of the Accept
        header MUST be application/cbor.

That's correct